Homeworks/Homework SQL/SQLQuery-Homework.sql

```diff
@@ -0,0 +1,339 @@
+ALTER TABLE Books
+DROP COLUMN AuthorFirstName;
+ALTER TABLE Books
+DROP COLUMN AuthorLastName;
+ALTER TABLE Books
+DROP COLUMN Type;
+
+CREATE TABLE Author
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+FirstName VARCHAR (50),
+LastName VARCHAR (50),
+Email VARCHAR (30),
+)
+
+CREATE TABLE TYPE
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Name VARCHAR (30),
+)
+
+ALTER TABLE Books
+ADD AuthorId INT REFERENCES Author(id);
+
+ALTER TABLE Books
+ADD TypeId INT REFERENCES Type(id);
+
+SELECT *
+FROM Books;
+
+ALTER TABLE Librarians
+DROP COLUMN IsOnHoliday;
```

Comment:
The change is here should be only on the type of the column because it can only hold two values (true / false) so a BIT data type would be enough without having to create a new table.